apps/charterafrica/src/lib/data/common/processPageOrganisations.js

```diff
@@ -0,0 +1,192 @@
+import {
+  organisations as orgMocks,
+  tools as toolMocks,
+} from "@/charterafrica/lib/data/_mock/ecosystemJson";
+import queryString from "@/charterafrica/utils/articles/queryString";
+import formatDateTime from "@/charterafrica/utils/formatDate";
+import labelsPerLocale from "@/charterafrica/utils/translationConstants";
+
+const orQueryBuilder = (fields, search) => {
+  return fields.map((field) => ({ [field]: { like: search } }));
+};
+
+async function processPageSingleOrganisation(page, api, context) {
+  const { params, locale } = context;
+  const { slug: collection } = page;
+  const slug = params.slugs[2];
+  const { docs } = orgMocks(collection, {
+    locale,
+    where: {
+      slug: {
+        equals: slug,
+      },
+    },
+  });
+  if (!docs?.length) {
+    return null;
+  }
+  const filterLabels = labelsPerLocale[locale];
+  const organisation = docs[0] || {};
+  const { docs: toolDocs } = toolMocks("tools", {
+    locale,
+    where: {
+      organisation: {
+        in: [organisation.id],
+      },
+    },
+  });
+
+  const tools = toolDocs.map((tool) => ({
+    ...tool,
+    image: tool.avatarUrl ?? null,
+    description: tool?.description || " ",
+    name: tool.name || " ",
+  }));
+
+  return {
+    ...page,
+    blocks: [
+      {
+        slug: "entity",
+        image: organisation.avatarUrl ?? null,
+        name: organisation?.name ?? null,
+        location: organisation?.country ?? null,
```

Comment:
Is this organisation's country or location? Mock data says location for organisation and  country for people

Reply:
We will use location here instead of country for consistency with the data team